lib/msal-common/src/authority/B2cAuthority.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Authority } from "./Authority";
+import { OpenIdConfigResponse } from "./OpenIdConfigResponse";
+import { AuthorityType } from "./AuthorityType";
+import { B2CTrustedHostList, Constants } from "../utils/Constants";
+import { INetworkModule } from "../network/INetworkModule";
+import { ClientConfigurationError } from "../error/ClientConfigurationError";
+
+/**
+ * The AadAuthority class extends the Authority class and adds functionality specific to the Azure AD OAuth Authority.
```

Comment:
All comments in this class mention AAD, but should instead be switched over to B2C.

Reply:
Addressed. Sorry missed a push. Now B2C file is upto date and accurate.